Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -36,6 +36,13 @@ public struct TestCommand: AsyncParsableCommand, HasTrackableParameters {
     )
     var clean: Bool = false
 
+    @Flag(
+        name: .shortAndLong,
+        help: "When passed, selective tests are not uploaded to the remote storage.",
```

Comment:
I recommend using "server" to align with the terminology that we are using in the docs:

```suggestion
        help: "When passed, the result necessary for test selection is not persisted to the server.",
```